src/Bridge/Symfony/Bundle/DependencyInjection/Compiler/GraphQlMutationResolverPass.php

```diff
@@ -31,6 +31,10 @@ final class GraphQlMutationResolverPass implements CompilerPassInterface
      */
     public function process(ContainerBuilder $container)
     {
+        if (!$container->getParameter('api_platform.graphql.enabled')) {
```

Comment:
You should move this check in [ApiPlatformBundle::build](https://github.com/api-platform/core/blob/be932e901402000adf54fa38e0b741f36474be79/src/Bridge/Symfony/Bundle/ApiPlatformBundle.php#L48), so the CompilerPass won't be executed (same with other GraphQl CompilerPasses)

Reply:
thanks @vincentchalamon I updated the PR, I did the same thing for elasticsearch.